packages/mattermost-redux/src/types/plugins.ts

```diff
@@ -134,3 +139,37 @@ export type MarketplacePlugin = { // TODO remove this in favour of the definitio
     manifest: PluginManifest;
     installed_version?: string;
 }
+
+export type PluginsState = {
```

Comment:
This is already defined in `/types/store/` `index.ts` and `plugins.ts` (not in redux). Not 100% sure the value of adding it here (but I like that it is more deeply typed than the available one).

Reply:
This actually also (at least partially) needs to go to the mobile repo as well, as it will also be using this to determine if the Apps plugin is currently available. It seems weird, but we are using the existing mechanism of enabling/disabling webapp plugins at runtime for the sole purpose of keeping states in sync with the Apps plugin's state.